roottest/root/treeformula/retobj/CMakeLists.txt

```diff
@@ -1,7 +1,33 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOT_GENERATE_DICTIONARY(G__TreeFormulaRetobjGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/Event.h
+  LINKDEF ${CMAKE_CURRENT_SOURCE_DIR}/EventLinkDef.h)
+
+ROOT_LINKER_LIBRARY(TreeFormulaRetobjGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/TreeFormulaRetobjGeneration.cxx
+  G__TreeFormulaRetobjGeneration.cxx
+  LIBRARIES Core Tree Hist MathCore)
```

Comment:
From https://github.com/root-project/roottest/pull/773#discussion_r2020254363:

Please add the generation of the dictionary as required fixture.